libvast/src/system/partition.cpp

```diff
@@ -551,7 +554,7 @@ passive_partition(caf::stateful_actor<passive_partition_state>* self, uuid id,
   // to queries. The `skip` default handler is used to buffer all messages
   // arriving until then.
   self->set_default_handler(skip);
-  self->send(caf::actor_cast<caf::actor>(fs), atom::read_v, path);
+  self->send(caf::actor_cast<caf::actor>(fs), atom::mmap_v, path);
```

Comment:
It's great to see that we literally only need a one-line change to switch to mmap'ing now!